src/main/java/com/coherentsolutions/training/automation/java/web/urnezaite/util/Log.java

```diff
@@ -0,0 +1,17 @@
+package com.coherentsolutions.training.automation.java.web.urnezaite.util;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Log {
+    private static final Logger logger = LogManager.getLogger(Log.class);
+
+    public static void info(String message) {
+        logger.info(message);
+    }
+
+    public static void error(String message) {
+        logger.error(message);
+    }
+}
```

Comment:
no need to create your own Logger, please take a look an example how to use logger https://mkyong.com/logging/apache-log4j-2-tutorials/